src/lib/api/helper/interceptors.ts

```diff
@@ -119,7 +119,7 @@ export function responseErrorInterceptor() {
             (error.response.status === 401 || error.response.status === 403)
         ) {
             ConfigStore.delete(CONFIG_KEYS.AUTH_TOKEN);
-            throw new CommandError(COMMON_LOG_MESSAGES.RequireAuth);
+            throw new CommandError(COMMON_LOG_MESSAGES.RequireAuth, error.response.status);
```

Comment:
We are checking AUTH_TOKEN before executing every CLI command, right ? @manishprajapati52